SQL/2016-01-22-CurrentlyProcessedRenamedInMRIUploadTable.sql

```diff
@@ -1,2 +1,2 @@
 ALTER TABLE mri_upload CHANGE `Processed` `InsertionComplete` tinyint(1) NOT NULL DEFAULT '0';
-ALTER TABLE mri_upload CHANGE `CurrentlyProcessed` `Inserting` tinyint(1) NOT NULL DEFAULT '0';
+ALTER TABLE mri_upload CHANGE `CurrentlyProcessed` `Inserting` tinyint(1) DEFAULT NULL;
```

Comment:
this will allow me to set more easily the condition for launching Loris-MRI cronjob (imaging_upload_fil_cronjob.pl)!